GAME_DESIGN_PRINCIPLES.md

```diff
@@ -0,0 +1,97 @@
+Cubyz' special sauce is its simplicity. Keeping everything simple on the surface makes the game approachable for beginners, while the hidden depth keeps it interesting for skilled players. Avoid features that would add unnecessary tedium to the player's experience. Always think about how it would effect the player's first impressions of the game.
+
+### Fuel the Player's Curiosity
+The world is filled with secrets, and we want the player to find these secrets on their own without any outside help or guide.
+
+### Problems have Multiple Solutions
+Problems faced in a particular age of progression should have multiple solutions, instead of just one solution for everything.
+For example, in the "Pre-Caves Age," the player will not have access to coal, as coal spawns low in the world and is shrouded in darkness. There's many solutions to this, however:
+- Find a dim, above ground light source.
+- Search for a cave with exposed coal.
+- Wander into the darkness to find a cave that's bright enough to mine in.
+- Find an above-ground structure that can grant the player coal.
+- Dig straight down until you find coal.
+
+Of course, this problem is completely solved as soon as the player gets coal, as they can now explore caves to find more coal with their newfound torches, but it's important that the player has these options in the first place so they can have more freedom. Not one option is better or easier than the other, and if one turns out to be, then it should be allowed to be adjusted to encourage other methods.
+
+### Explain when Needed
+If something progress-related has no obvious explanation, the player will have to rely on a wiki to find out how to progress. One example is the bellows; players won't know to place it next to a furnace and jump on it repeatedly. The issue can be fixed by adding a tooltip that tells them how it works, while leaving out details they can intuitively find out themselves.
+
+### Micro Moments
+These are tiny things the player does in between larger events; examples include:
+- Travelling
+- Parkour
+- Mining
+- Building
+- Crafting
+- Managing inventory
+- Fighting
+
+These moments are extremely important as they let the player mentally rest, so making sure they're as satisfying and consistent as possible is a must.
+
+# Balancing
+When balancing the game, keep in mind how players might interact with the world, the wildlife, and each other.
+
+### 2OP4ME
+At no point should the player be extremely hard to kill. Armor, tools, accessories, and buffs should be designed around aiding the player in skill-based encounters, not making them win regardless of skill.
+
+### Trade-offs
+If the player is given something to aid them, then it should have an appropriate take-away to balance it.
+
+**Examples:**
+- Using rare resources to create a strong tool.
+- An accessory that heals the player, but takes away energy or some other resource.
+- Armor that grants a ton of defense, but makes the player slow.
```

Comment:
Armor should never make the player slow, at least not in terms of movement speed.

Reply:
I would suggest less concrete descriptions of mechanics which are still in development and we're not reality checked. Instead of giving examples poor some water and make the description softer. Don't say you will be slow, say that there will be a trade-off, that progression is horizontal.